README.md

```diff
@@ -3,3 +3,16 @@
 
 
 This is the codebase for the Colligo application, showcasing digital manuscripts.
+
+## Local Development Setup
+1. Clone and run `bundle install`
+2. Run `bin/rake db:migrate RAILS_ENV=development` or an equivalent to populate the database with development data
+3. Run `bundle exec rake` to download and install supplementary service software (Jetty) and run tests
+4. Run `bundle exec rake jetty:start`
+5. Once the Jetty service has finished spinning up, run `rake colligo:fixtures` to fill in solr fixtures (as distinct for DB migration above)
```

Comment:
Everything up to here can be done initially using the `rake colligo:install` task on the first install.  Not sure if we want to call out that install task separately.